Makefile

```diff
@@ -0,0 +1,34 @@
+
+
+test: test.unit test.integration
```

Comment:
This is the first obvious candidate for removal or extraction. I have not found a single place in the osal from where I can run ALL unit and integration tests. I suspect that there is some private branch of `nasa/osal` where such script exists.

Having such a one-click solution for running all of the tests is essential for making sure that the osal is not broken when making changes and testing them on both Linux and macOS.

Please advise if I should remove this custom `Makefile` and rather use some existing test scripts.

Reply:
Looks like this will  satisfy it: https://github.com/nasa/osal/pull/403.